.github/workflows/ci.yml

```
@@ -13,6 +13,9 @@ concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
   cancel-in-progress: true
 
+permissions:
```

actionlint (1.7.9)

key "permissions" is duplicated in workflow. previously defined at line:3,col:1

(syntax-check)